packages/amplify_datastore/CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## 0.0.1-dev.5 (2020-12-31)
+
+### Bug Fixes
```

Comment:
The API inclusion is more a feature, and some of the other changes aren't exactly bug fixes either. Maybe just: 
```suggestion
```